ajax/libs/reactive-elements/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "reactive-elements",
+  "version": "0.6.7",
+  "description": "Allows to use React.js component as custom element",
+  "filename": "reactive-elements.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PixelsCommander/ReactiveElements.git"
+  },
+  "keywords": [
+    "react",
+    "web-components",
+    "custom elements",
+    "reactjs"
+  ],
+  "author": {
+    "name": "Denis Radin aka PixelsCommander",
+    "email": "denis.radin@gmail.com",
+    "url": "http://pixelscommander.com/"
```

Comment:
Why @clairetsai818 just updated the PR without any response comment to the review? Is this kind of "Teacher says"?